### PR TITLE
Fix incorrect comments in IsPodAvailable

### PR DESCRIPTION
**Pull request by janetkuo**
@kubernetes/sig-config 

[Analytics]()

### CODE REVIEW

File: pkg/util/deployment/deployment.go [hunk 1]
```diff
@@ -286,8 +286,8 @@ func IsPodAvailable(pod *api.Pod, minReadySeconds int) bool {
 	for _, c := range pod.Status.Conditions {
 		// we only care about pod ready conditions
 		if c.Type == api.PodReady && c.Status == api.ConditionTrue {
-			// 2 cases that this ready condition is valid (passed minReadySeconds, i.e. the pod is ready):
-			// 1. minReadySeconds <= 0
+			// 2 cases that this ready condition is valid (passed minReadySeconds, i.e. the pod is available):
+			// 1. minReadySeconds <= 0, or
```
Comment (kargakis) [inline pkg/util/deployment/deployment.go]: By validation, this is always non-negative: https://github.com/kubernetes/kubernetes/blob/ee2a0694b649941fc0c3be606746db041b75b91d/pkg/apis/extensions/validation/validation.go#L324

So it should be "minReadySeconds == 0"

Comment (janetkuo) [inline pkg/util/deployment/deployment.go]: Thanks. Comment addressed.

**Comment by kargakis** [comment]
lgtm

**Comment by janetkuo** [comment]
Squashing.

**Pull request merged**